wordpress-mobile/WordPress-Android
# [Gutenberg] Audio Block - Audio Player UI

`Gutenberg Mobile` https://github.com/wordpress-mobile/gutenberg-mobile/pull/3037
`Gutenberg` https://github.com/WordPress/gutenberg/pull/27467

## Solution
Showcases the Audio Player UI with Consolidated Media Picker for file picking. 

## Testing

1. Create a new post. 
2. Add an Audio Block. 
3. Add an audio file from the device to see upload in progress and success state. 
4. Add an audio file from the device while in airplane mode to see the error state. 

### Success
Light Mode | Dark Mode 
--------|-------
 <kbd><img src="https://user-images.githubusercontent.com/1509205/105439449-e1b83c80-5c32-11eb-850f-d1ab1306e6ca.png" width="320"></kbd>  |    <kbd><img src="https://user-images.githubusercontent.com/1509205/105439447-e0870f80-5c32-11eb-996c-74af4c6c1357.png" width="320"></kbd>
   
### Uploading

Light Mode | Dark Mode 
--------|-------
<kbd><img src="https://user-images.githubusercontent.com/1509205/105439444-df55e280-5c32-11eb-839f-b777868db8cc.png" width="320"></kbd>   |    <kbd><img src="https://user-images.githubusercontent.com/1509205/105440539-b898ab80-5c34-11eb-8878-622037a2a3d1.png" width="320"></kbd>

### Error

Light Mode | Dark Mode 
--------|-------
 <kbd><img src="https://user-images.githubusercontent.com/1509205/105441337-53de5080-5c36-11eb-861a-e9915a1160b9.png" width="320"></kbd>   |    <kbd><img src="https://user-images.githubusercontent.com/1509205/105441334-52ad2380-5c36-11eb-9ce2-a082fd83e253.png" width="320"></kbd>

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
